src/fluka/fluka_funcs.cpp

```diff
@@ -984,7 +985,7 @@ void fludag_all_tallies(std::ostringstream& mstr, std::map<std::string, pyne::Ta
   std::list<std::string> tally_parts;
   std::string tally_id;
   for (it = tally_map.begin() ; it != tally_map.end() ; ++it) {
-    tally_id = (it->second).tally_type + "/" + (it->second).particle_name;
+    tally_id = (it->second).tally_type + "/" + (it->second).particle_names[0];
```

Comment:
Originally a pyne::Tally object had a single particle that described it, I take it from the new name that they now have multiple names?

Reply:
That was my understanding too, yeah. Though from these lines in pyne it looks like only single-particle tallies are intended for FLUKA, so maybe what we have here is ok?

https://github.com/pyne/pyne/blob/010c3d3bd2b1530aea4d7e00f2771a129a028673/src/tally.cpp#L600-L603

Hoping for @bam241's thoughts here as it looks like was part of that update.